### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by asad11ali**
-This sections of the mobile
website should include:
○ "Works" section
○ the "About myself" section (placed after the projects and before the
contact form):
-Make sure the content of the "About myself" section is aligned with flexbox.
-In order to lay out the elements in "Works" section you should use CSS Grid and
Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main
container that contains all of the cards and use Flexbox to lay out an individual
card.
-Please use ul and li for categories in "Work" section. Html, CSS, and JavaScript
should be list items.
-Please use ul and li for categories in "About myself" section. Languages,
Frameworks, and Skills should be list items.
-You don't need to implement the detail modal window that requires JavaScript.
-You must stick to the design as much as possible (e.g., font, colors, images, text,
margins) using the templates in Figma.
-Implement the button interactions (enable, hover, etc.).
-The website should be responsive, following these breakpoints:
○ Minimum mobile screen size should be 375px
○ Minimum desktop screen size should be 768px
-Use responsive values in your CSS rules, like percentages (i.e. width: 80%),
instead of pixels

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,9 +40,141 @@ <h4 class="sub-heading">
...
+         <span class="card-item">
+             <div>HTML</div>
+             <div>Bootstrap</div>
+             <div>Ruby</div>
+         </span>
+         <button class="btn-2">See project</button>
+     </div>
+     <div class="block2">
+        <h1>Profesional Art <br>
+         Printing Data</h1>
```
Comment (Pamphilemkp) [inline index.html]: It is allow to only have one h1 tag per page please consider refractoring your code use other heading tags: h2,h3,h4,h5,h6

File: index.html [hunk 2]
```diff
@@ -40,9 +40,141 @@ <h4 class="sub-heading">
...
+             <div>Bootstrap</div>
+             <div>Ruby</div>
+         </span>
+         <button class="btn-2">See project</button>
+     </div>
+     <div class="banner-2">
+        <div class="main-1">
+            <h1 class="heading-2">
+                About me
+            </h1>
```
Comment (Pamphilemkp) [inline index.html]: Please remove all the exciding h1 heading in this page as It is allow to only have one h1 tag per page please consider refractoring your code use other heading tags: h2,h3,h4,h5,h6

File: index.html [hunk 3]
```diff
@@ -40,9 +40,141 @@ <h4 class="sub-heading">
...
+         <span class="card-item">
+             <div>HTML</div>
+             <div>Bootstrap</div>
+             <div>Ruby</div>
+         </span>
+         <button class="btn-2">See project</button>
+     </div>
+     <div class="block2">
+        <h1>Profesional Art <br>
+         Printing Data</h1>
```
Comment (Pamphilemkp) [inline index.html]: Replace this h1 heading with another one as It is allow to only have one h1 tag per page please consider refractoring your code use other heading tags: h2,h3,h4,h5,h6

**Review by Pamphilemkp — CHANGES_REQUESTED** [review]
## Hi :raising_hand: @asad11ali  , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!




## Highlights :green_circle: 


- PR is descriptive and professional :+1: 
- No Linter error :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions



_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy] have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can contact us via._

**Review by basitali111 — APPROVED** [review]
# Approve 🟢 🟢 🟢

Hi @asad11ali 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉



giphy

Highlights
No linter errors ✅
Good Pr title and description 💪🏿
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Kindly make use of the provided README.md template provided in your dashboard otherwise the next project will be marked as invalid
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @basitali111 

_As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion by contacting us.

**Pull request merged**